includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -676,6 +676,10 @@ public function remove_all_actions_on_preflight_check( $response, $handler, $req
 			remove_all_actions( 'woocommerce_store_api_checkout_order_processed' );
 			// Avoid increasing coupon usage count during preflight check.
 			remove_all_actions( 'woocommerce_order_status_pending' );
+
+			// Avoid creating new accounts during preflight check.
+			remove_all_filters( 'woocommerce_checkout_registration_enabled' );
+			remove_all_filters( 'woocommerce_checkout_registration_required' );
```

Comment:
I don't know if any difference but can you share why do you choose to remove the filter not disable the filter by passing `false`?

Reply:
WooCommerce Memberships uses a `9999` priority (third argument of `add_filter`), other plugins can use even higher priorities, so it's better to just remove all filters.

Reply:
Thanks for sharing 👍 It does work so I guess we don’t need to use `PHP_INT_MAX` to priority and return `false`.